config/openiotsdk/cmake/sdk.cmake

```diff
@@ -134,14 +144,14 @@ if(TARGET mbedtls-config)
             ${OPEN_IOT_SDK_CONFIG}/mbedtls
     )
 
-    target_sources(mbedtls-config 
+    target_sources(mbedtls-config
         INTERFACE
             ${OPEN_IOT_SDK_CONFIG}/mbedtls/platform_alt.cpp
     )
 
     target_compile_definitions(mbedtls-config
         INTERFACE
-            MBEDTLS_CONFIG_FILE="mbedtls_config.h"
+            MBEDTLS_CONFIG_FILE="${OPEN_IOT_SDK_CONFIG}/mbedtls/mbedtls_config.h"
```

Comment:
I think we don't need this. We already included the `${OPEN_IOT_SDK_CONFIG}/mbedtls` directory to `mbedtls-config` target.

Reply:
Oh, I see the reason :)

Reply:
I can revert and double check it still builds properly.